website_supplier_list/views/website_supplier.xml

```diff
@@ -0,0 +1,195 @@
+                            <a class="pull-left" t-attf-href="/suppliers/#{slug(partner)}"
+                               t-field="partner.image_small"
+                               t-field-options='{"widget": "image", "class": "media-object"}'
+                            ></a>
+                            <div class="media-body" style="min-height: 64px;">
+                                <a class="media-heading" t-attf-href="/suppliers/#{slug(partner)}">
+                                    <span t-field="partner.display_name"/>
+                                </a>
+                                <div t-field="partner.website_short_description"/>
+                            </div>
+                        </div>
+                    </t>
+                        </div>
+                    </div>
+
+                </div>
+            </div>
+            <div class="oe_structure"/>
+        </div>
+    </t>
+</template>
+
+<!-- Option: left column: World Map -->
+<template id="opt_country" inherit_id="website_supplier_list.index" customize_show="True" name="Show Map">
+    <xpath expr="//div[@id='ref_left_column']" position="inside">
+        <!-- modal for large map -->
+        <div class="modal fade customer_map_modal" tabindex="-1" role="dialog" aria-labelledby="myLargeModalLabel" aria-hidden="true">
+          <div class="modal-dialog modal-lg">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <button type="button" class="close" data-dismiss="modal" aria-hidden="true">×</button>
+                    <h4 class="modal-title">World Map</h4>
+                </div>
+                <iframe t-attf-src="/google_map/?width=898&amp;height=485&amp;partner_ids=#{ google_map_partner_ids }&amp;partner_url=/suppliers/"
+                style="width:898px; height:485px; border:0; padding:0; margin:0;"></iframe>
+            </div>
+          </div>
+        </div>
+        <!-- modal end -->
+        <h3>World Map<button class="btn btn-link" data-toggle="modal" data-target=".customer_map_modal"><span class="fa fa-external-link" /></button></h3>
+        <ul class="nav">
+            <iframe t-attf-src="/google_map?width=260&amp;height=240&amp;partner_ids=#{ google_map_partner_ids }&amp;partner_url=/suppliers/"
+                style="width:260px; height:240px; border:0; padding:0; margin:0;" scrolling="no"></iframe>
+        </ul>
+    </xpath>
+</template>
+
+<template id="opt_country_list" inherit_id="website_supplier_list.index" customize_show="True" name="Filter on Countries">
+    <xpath expr="//div[@id='ref_left_column']" position="inside">
+        <h3>References by Country</h3>
+        <ul class="nav nav-pills nav-stacked mt16 mb32">
+            <t t-foreach="countries" t-as="country_dict">
+                <t t-if="country_dict['country_id']">
+                    <li t-att-class="country_dict['country_id'][0] == current_country_id and 'active' or ''">
```

Comment:
I think you can remove the `or ''` part. Qweb removes attributes if its result is `False`.